src/index.ts

```diff
@@ -47,20 +51,26 @@ export async function startNotarize(opts: NotarizeStartOptions): Promise<Notariz
     }
     d('zip succeeded, attempting to upload to apple');
 
+    const notarizeStartOpts = [
```

Comment:
Can we call this args instead of opts?
```suggestion
    const notarizeArgs = [
```